### PR TITLE
fix the hamburger position and consume fix for pop up

### PR DESCRIPTION
**Pull request by alex-bezek**
### Summary
Hamburger position should maintain relative to its normal flow to keep within the main content and get its padding. But it should be bumped over slightly without affecting flow

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-consumer-layout/src/Layout.scss [hunk 1]
```diff
@@ -123,10 +123,9 @@
   .nav-burger {
     display: block;
     font-size: 20px;
-    left: 15px;
-    padding: 0;
-    position: absolute;
-    top: 10px;
+    left: -13px;
```
Comment (mhemesath) [inline packages/terra-consumer-layout/src/Layout.scss]: where is 13px coming from?

File: packages/terra-consumer-nav/src/components/modal/Modal.scss [hunk 2]
```diff
@@ -30,38 +30,37 @@
...
-    border: 0;
-    color: var(--terra-consumer--modal-close-button-color, #fff);
-    font-size: 16px;
+  .close-button-container {
     margin: 0;
     padding: 0;
     position: absolute;
     right: 20px;
-    top: 5px;
+    top: 0;
```
Comment (amf9t2) [inline packages/terra-consumer-nav/src/components/modal/Modal.scss]: what's difference between not adding `top` and  adding `top: 0` ?

File: packages/terra-consumer-nav/src/components/user-profile/ProfileLinks.jsx [hunk 3]
```diff
@@ -53,13 +52,13 @@ class ProfileLinks extends React.Component {
         const toggleIcon = isOpen ? <IconChevronUp /> : <IconChevronDown />;
         return (
           <div key={linkItem.text} className={cx(isOpen && 'open')}>
-            <Button onClick={() => { this.handleToggle(index); }} className={cx('link', 'toggler-wrapper')}>
+            <button onClick={() => { this.handleToggle(index); }} className={cx('link', 'toggler-wrapper')}>
               <Arrange
                 align="stretch"
```
Comment (amf9t2) [inline packages/terra-consumer-nav/src/components/user-profile/ProfileLinks.jsx]: align `top` didn't fix the icon alignment  ?

File: packages/terra-consumer-nav/src/components/modal/Modal.scss (hunk 2 above)
Comment (alex-bezek) [reply packages/terra-consumer-nav/src/components/modal/Modal.scss]: removed it

**Pull request merged**